tests/test_spark.py

```diff
@@ -169,6 +216,17 @@ def _require_spark():
     _wait_for_spark()
 
 
+def _require_spark_cli():
```

Comment:
Is this needed for the soak tests?  I'd rather have the test runner code be responsible for setting up the environment (incuding the CLI), rather than the tests themselves.